src/index.html

```diff
@@ -4,13 +4,38 @@
     <meta charset="UTF-8">
     <meta
       name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+      content="width=device-width, initial-scale=1.0"
     >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <title>Search bar airbnb</title>
+    <title>Чорновик</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+
+    <form 
+      action="*" 
+      method="post"
+      data-qa="big"
+      >
+      <input 
+        type="search" 
+        placeholder="Try &quot;Los Angeles&quot; " 
+        data-qa="keypress"
+        class="big_input"
+      >
+    </form>
+
+    <form 
+      action="*" 
+      method="post"
+      data-qa="small"
+    >
+      <input 
+        type="search" 
+        placeholder="Try &quot;Los Angeles&quot;" 
+        class="small_input"
```

Comment:
```suggestion
        class="input input--small"
```

Reply:
it should work like this

Reply:
Hi, can you help me please to deal with my mistakes. I don`t use other styles except those that were in the mockup.
![image](https://user-images.githubusercontent.com/115538688/203056741-f0d432c0-08f4-4580-aba5-57d82357d4e9.png)
![image](https://user-images.githubusercontent.com/115538688/203056875-ae379a04-0895-441c-bdcc-027daf6bef34.png)